.circleci/config.yml

```
@@ -24,7 +24,8 @@ jobs:
   check_sepolia_rpc_endpoints:
     circleci_ip_ranges: true
     docker:
-      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.35.0
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.47.3
```

Let's update the pipeline parameter above (it's currently 0.47.0) and use that for each job
```suggestion
      - image: <<pipeline.parameters.ci_builder_image>>
```